packages/main/src/Tokenizer.ts

```diff
@@ -465,35 +485,34 @@ class Tokenizer extends UI5Element {
 		tokens.forEach(token => { token.selected = !tokensAreSelected; });
 	}
 
-	_handleTokenSelection(event, deselectAll = true) {
-		if (event.target.hasAttribute("ui5-token")) {
-			const deselectTokens = deselectAll ? this._tokens : [event.target];
+	_handleTokenSelection(e: KeyboardEvent | MouseEvent, deselectAll = true) {
+		const target = e.target as Token;
+		if (target.hasAttribute("ui5-token")) {
+			const deselectTokens = deselectAll ? this._tokens : [];
 
 			deselectTokens.forEach(token => {
-				if (token !== event.target) {
+				if (token !== target) {
 					token.selected = false;
 				}
 			});
 		}
 	}
 
-	_fillClipboard(shortcutName, tokens) {
+	_fillClipboard(shortcutName: ClipboardDataOperation, tokens: Array<Token>) {
 		const tokensTexts = tokens.filter(token => token.selected).map(token => token.text).join("\r\n");
 
 		/* fill clipboard with tokens' texts so parent can handle creation */
-		const cutToClipboard = event => {
-			if (event.clipboardData) {
-				event.clipboardData.setData("text/plain", tokensTexts);
-			} else {
-				event.originalEvent.clipboardData.setData("text/plain", tokensTexts);
+		const cutToClipboard = (e: ClipboardEvent) => {
+			if (e.clipboardData) {
+				e.clipboardData.setData("text/plain", tokensTexts);
 			}
 
-			event.preventDefault();
+			e.preventDefault();
 		};
 
-		document.addEventListener(shortcutName, cutToClipboard);
+		document.addEventListener(shortcutName, cutToClipboard as EventListenerOrEventListenerObject);
```

Comment:
these casts to EventListenerOrEventListenerObject are redudant

Reply:
Done